packages/suite-analytics/src/types/definitions.ts

```diff
@@ -0,0 +1,19 @@
+import { AppUpdateEventStatus } from '../constants';
+
+export type OnboardingAnalytics = {
```

Comment:
used anywhere else outside this package?

Reply:
In `suite` for in redux state